services/package-manager/src/test/java/org/openthinclient/pkgmgr/it/PackageInstallTest.java

```diff
@@ -56,7 +58,44 @@ public void testInstallSinglePackage() throws Exception {
     assertTrue(fooPackage.isPresent());
     assertTrue(packageManager.install(Collections.singletonList(fooPackage.get())));
 
-    // verify the existence files from the package.
+    File fooXml = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo.xml");
+    File fooSample = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo-tiny.xml.sample");
+    File fooSfs = new File(configuration.getInstallDir().getPath() + File.separator + "sfs" + File.separator + "package" + File.separator + "foo.sfs");
+
+    assertTrue("foo.xml not installed",fooXml.exists());
+    assertTrue("foo-tiny.xml.sample not installed",fooSample.exists());
+    assertTrue("foo.sfs not installed",fooSfs.exists());
+    
+    assertEquals(0,configuration.getTestinstallDir().listFiles().length);
+  }
+  
+  @Test
+  public void testInstallSinglePackageDependingOther() throws Exception {
+    final DPKGPackageManager packageManager = preparePackageManager();
+
+    final Optional<org.openthinclient.util.dpkg.Package> bar2Package = packageManager.getInstallablePackages()
+            .stream()
+            .filter(pkg -> pkg.getName().equals("bar2"))
+            .findFirst();
+
+    assertTrue(bar2Package.isPresent());
+    assertTrue(packageManager.install(Collections.singletonList(bar2Package.get())));
+
+    File fooXml = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo.xml");
```

Comment:
These File-construction lines are not readable. Please change that to something like the following:

```
    final Path installDirectory = configuration.getInstallDir().toPath();
    
    Path fooXml = installDirectory.resolve("schema").resolve("application").resolve("foo.xml");
```

Whether or not the files exist can be verified using the following:

```
    Files.isRegularFile(fooXml);
    Files.exists(fooXml);
```

This will make the code overall more readable.